bats/scripts/package.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+set +o xtrace
+
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
+
+git submodule update --init --recursive
+
+if [ ! -f bats-core/bin/bats ]; then
+    echo "This doesn't look like the rancher-desktop/bats directory: $PWD"
+    exit 1
+fi
+
+# Bundle jq for linux and macos
+
+rm -rf bin/
+mkdir -p bin/darwin
+mkdir -p bin/linux
+
+JQ_VERSION=1.6
+URL=https://github.com/stedolan/jq/releases/download/
+
+wget --no-verbose $URL/jq-$JQ_VERSION/jq-osx-amd64 -O bin/darwin/jq
+chmod +x bin/darwin/jq
+
+wget --no-verbose $URL/jq-$JQ_VERSION/jq-linux64 -O bin/linux/jq
+chmod +x bin/linux/jq
```

Comment:
Should we be using these for the tests?

Reply:
> Should we be using these for the tests?

Yes, I was going to put this at the bottom of `bats/tests/helpers/load.bash` (so after `os.bash` has been loaded):

```shell
export PATH="$topdir/bin/${OS/windows/linux}:$PATH"
```

I was going to do this in a separate PR, but I can add it here instead, if I have to make other changes anyways. 😄